src/pages/search/CristinSearchPagination.tsx

```diff
@@ -24,6 +24,7 @@ export const CristinSearchPagination = ({ totalCount }: CristinSearchPaginationP
 
   return (
     <TablePagination
+      aria-live="assertive"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#live_regions
> Normally, only aria-live="polite" is used

```suggestion
      aria-live="polite"
```